Wikipedia/Code/TabBarDelegate.swift

```diff
@@ -0,0 +1,13 @@
+//
+//  TabBarDelegate.swift
+//  Wikipedia
+//
+//  Created by Matt Cl on 4/13/20.
+//  Copyright © 2020 Wikimedia Foundation. All rights reserved.
+//
```

Comment:
We take off our headings for new files in this project. I've just developed the habit of removing them each time but I think @joewalsh had found an automated way a while back. I also found [this](https://oleb.net/blog/2017/07/xcode-9-text-macros/) but it results in a single '//' header, so not ideal but better than nothing.